src/main/PDF/PdfMessage.java

```diff
@@ -16,11 +16,11 @@ public enum PdfMessage implements Message {
   NO_SUCH_FILE("NO_SUCH_FILE;PDF does not exist"),
   ENCRYPTION_ERROR("ENCRYPTION_ERROR;Error encrypting/decrypting"),
   CROSS_ORG_ACTION_DENIED(
-      "CROSS_ORG_ACTION_DENIED:You are trying to modify another organization's pdf."),
+      "CROSS_ORG_ACTION_DENIED;You are trying to modify another organization's pdf."),
```

Comment:
im confused on this change?

Reply:
The error handler tries to parse based on ";", but when we have a ":" here it will give an internal server error instead of returning the actual error message, as it fails to parse